### DISCUSSION TITLE
第1支100%收益持股诞生！

### DISCUSSION THREAD

**Discussion by axiaoxin** [axiaoxin-com/investool#33] [Show and tell]
今天通过x-stock筛选并买入持有的一支票持仓浮盈实现翻倍，最高达到102+%，特此记录！

![039AA9C3-B41E-4101-8F2A-64DE50427F5C](https://user-images.githubusercontent.com/2876405/139454487-4f2db908-3b90-49b0-a57b-3ce9e944667d.jpeg)

**Comment by mojocn** [comment]
什么股票 大佬

**Reply by axiaoxin** [reply]
603606

**Reply by qiqisteve** [reply]
x-stock 是什么啊

**Reply by axiaoxin** [reply]
> x-stock 是什么啊

就是这个项目investool，改名了

**Reply by qiqisteve** [reply]
谢谢大佬

**Reply by qiqisteve** [reply]
大佬，最近有啥好的股推荐吗

**Reply by axiaoxin** [reply]
> 大佬，最近有啥好的股推荐吗

抱歉，不做任何推荐

**Reply by qiqisteve** [reply]
> > 大佬，最近有啥好的股推荐吗
> 
> 抱歉，不做任何推荐

留下了没有技术的泪水.jpg
